EULA.md

```diff
@@ -1,336 +1,139 @@
+
+The Provider reserves the right at any time to modify this Agreement and to impose new or additional terms or conditions on your use of the Services. Any material modifications and additional terms and conditions will be communicated to you and, if accepted, will be effective immediately and will be incorporated into this Agreement. In the event that you refuse to accept such changes, the Provider will have the right to terminate this Agreement pursuant to the Section below titled “Termination of the Agreement”.
 
 ## Termination of the Agreement
 
-This Agreement may be terminated by you at any time by discontinuing your use
-of the Service and deleting your User Content or by written notice to the
-Provider's technical/support contact listed in the [list of
-providers](#providers). In such an event, the Provider may immediately
-terminate your account, but it may take up to 10 business days to do so. You
-should ensure prior to termination that you have transferred all data stored on
-the Services to another storage medium.
-
-This Agreement may be terminated by the Provider for any reason upon 48 hours
-notice to you. In such an event, you will be afforded a reasonable opportunity
-to retrieve your User Content from the Services. If your account access has
-been suspended pursuant to [Enforcement](#enforcement), you may request to retrieve your User
-Content from the Services by emailing Provider's technical/support contact
-listed in the [list of providers](#providers). The Provider will take
-reasonable steps to arrange for you to retrieve such content, provided
-retrieval does not require an unreasonable amount of Provider's time or
-resources and that the content is not illegal; is not unlawfully pornographic
-or indecent; does not depict extreme acts of violence; does not advocate
-bigotry or hatred against any person or group of people based on their race,
-religion, ethnicity, sex, gender identity, sexual preference, disability, or
-impairment; and does not violate the privacy or infringe the rights of others).
-Any such retrieval shall be at your own expense.
+This Agreement may be terminated by you at any time by discontinuing your use of the Service and deleting your User Content or by written notice to the Provider’s technical/support contact listed here. In such an event, the Provider may immediately delete your account, but it may take up to four (4) weeks to do so. You should ensure prior to termination that you have transferred all data stored on the Services to another storage medium.
```

Comment:
Does this mean we always need to delete all user data within 4 weeks of them requesting termination?

Reply:
Yes, increased from 10 days in prior version